### ISSUE TITLE
Long-term cookie values are not rotated, can create new users randomly

### ISSUE THREAD

**Issue by sveeke** [SURFnet/learning-analytics#150]
In `AppBundle/Manager/UserManager.php`, the `identifyUser` method loads a user from a cookie value. The cookie has a fixed name (`lauid`), uses a standard UUID4 for its content (based on a properly random value, not timing or IP). It is valid for a year but lacks any mechanism for rotating or regenerating, making it a good target for stealing. If an unknown cookie value is provided, a new user entity is created and persisted to the database!

This all sounds like a bad idea - and the code says `@todo: cookies should be waayy more secure!`!

It may be that the SAML layer prevents such issues before it gets to this point. There is clearly already awareness of the weaknesses here.

**Comment by pablothedude** [comment]
Gaat dit nog een probleem zijn als users maar 3 maanden gebruikt gaan worden? Zie #155.
Ik wil wel een oplossing maken om de uid te verversen maar als dat binnen de drie maanden zal zijn heeft dit denk ik niet veel zin?

**Comment by hermanvand** [comment]
Ja, dat lijkt me ook. Als we de uid weggooien hoeven we hem uiteraard niet te verversen.
